sound/soc/sof/intel/hda-trace.c

```diff
@@ -19,6 +19,9 @@
 #include "../ops.h"
 #include "hda.h"
 
+#define CREATE_TRACE_POINTS
```

Comment:
hda-trace is also firmware tracing. The correct place is in the sound/soc/sof/core.c, imho
or sound/soc/sof/debug,c , but core is tad better, @plbossart what is your take on it?

Reply:
@nklayman, I still think this is also a wrong location, sof/core.c or sof/debug.c would be better.

Especially that you are adding trace points to core code and sof/intel/hda-trace.c is seldom compiled for ARM.

Reply:
It would have to go in `sof/intel/[something]`, as all the tracepoints defined in `trace/events/sof_intel.h` are specific to the intel code. The core tracepoints are all defined in `trace/events/sof.h`, which in included in `sof/core.c`. Any suggestions for that directory?

Reply:
Ah, OK.
I would then add this to the sof/intel/hda.c, the hda-trace.c is really for firmware tracing. At some point we might introduce a common, slim 'core' c file for Intel and move there?